css/style.css

```diff
@@ -0,0 +1,2927 @@
+}
+
+.header__menu-link:hover,
+.header__menu-link:focus,
+.header__menu-link:active {
+  width: 60px;
+  height: 2px;
+  color: #e51937;
+  border-bottom: 2px solid #e51937;
+}
+
+.header__burger {
+  padding-right: 0;
+  padding-top: 3px;
+  padding-left: 3px;
+}
+
+.top {
+  text-align: center;
+  background: url(../img/tesla-fon.png)  no-repeat;
+  background-position: 50% 0;
+  background-size: 100%;
+}
+
+.top__title {
+  max-height: 100px;
+  margin-top: 60px;
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: 0;
+}
+
+.top__subtitle {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.top__img {
+  max-width: 100%;
+  height: 450px;
+  margin-top: 25px;
+  margin-right: 75px;
+}
+
+.top__wrapper {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+  margin-left: 17px;
+  margin-bottom: 0;
+}
+
+.top__wrapper-charset {
+  display: flex;
+  align-items: center;
+  margin-right: 0;
+  margin-left: 3px;
+  margin-top: 4px;
+}
+
+.top__wrapper-item {
+  display: flex;
+  align-items: center;
+  margin-right: 100px;
+}
+
+.charset {
+  display: flex;
+  align-items: center;
+  margin-bottom: 5px;
+}
+
+.charset__item {
+  padding-right: 0;
+  padding-left: 0;
+  margin-bottom: 17px;
+}
+
+.charset__speed {
+  width: 46px;
+  height: 34.13px;
+  margin-right: 8px;
+  margin-left: 103px;
+  margin-top: 7px;
+  margin-bottom: 2px;
+}
+
+.charset__road {
+  margin-left: 25px;
+  margin-top: 15px;
+  margin-right: 0;
+}
+
+.charset__first {
+  display: flex;
+  align-items: baseline;
+  text-align-last: left;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  color: #000;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.charset__first-big {
+  height: 40px;
+  padding-top: 0;
+  padding-right: 7px;
+  margin-bottom: 0;
+  padding-bottom: 0;
+  padding-left: 6px;
+  margin-top: 0;
+  margin-right: 0;
+}
+
+.charset__first-small {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  padding-right: 50px;
+  padding-bottom: 0;
+}
+
+.charset__second {
+  font-size: 12px;
+  line-height: 15px;
+  text-align: left;
+  margin-bottom: 2px;
+  padding-left: 6px;
+}
+
+.button__outline {
+  border: none;
+  margin-bottom: 15px;
+  padding-right: 25px;
+  padding-bottom: 6px;
+  margin-left: 23px;
+  margin-right: 9px;
+}
+
+.button__outline-img {
+  padding-right: 0;
+  margin-right: 7px;
+  margin-left: 2px;
+  margin-bottom: 2px;
+}
+
+.button__outline:hover {
+  color: #e51937;
+}
+
+.button__text {
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  max-width: 90px;
+  text-align: center;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 16px;
+  margin-right: 0;
+  padding-bottom: 3px;
+}
+
+.button {
+  display: flex;
+  align-items: center;
+  padding: 10px 28px;
+  border: 2px solid #fff;
+  font-size: 13px;
+  text-transform: uppercase;
+  background-color: #fff;
+}
+
+.button__red {
+  border: 2px solid #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)